### ISSUE TITLE
Create develop branch

### ISSUE THREAD

**Issue by DaniilKl** [zarhus/meta-zarhus#16]
We should create `develop` branch so not to merge to `main` directly. Maybe this should be solved during https://github.com/zarhus/docs/pull/5.

**Comment by tym2k1** [comment]
[`develop` branch added](https://github.com/zarhus/meta-zarhus/tree/develop)

**Comment by DaniilKl** [comment]
Have you restricted pushing directly into `main`?

**Comment by tym2k1** [comment]
No, I don't have such access rights. Someone with access should then additionally restrict pushing to `main` and accepting PR's that target `main`. Perhaps @macpijan?
